src/main/target/STM32_UNIFIED/target.h

```diff
@@ -140,10 +152,16 @@
 #define TARGET_IO_PORTE 0xffff
 #define TARGET_IO_PORTF 0xffff
 
-#elif defined(STM32G47X)
+#elif defined(STM32G47X) || defined(STM32G47X_SPIRX)
+#if defined(STM32G47X)
 #define TARGET_BOARD_IDENTIFIER "SG47"
 
 #define USBD_PRODUCT_STRING     "Betaflight STM32G47x"
+#else
+#define TARGET_BOARD_IDENTIFIER "SF7S"
```

Comment:
Ahh, what's this? ``SG4S`` I guess?

Reply:
You always find my problems. Sorry fixed in actual PR.

Reply:
I like your code so i wanna check it closely.